src/classes/club.py

```diff
@@ -13,7 +19,28 @@ def __str__(self):
         return f"Club with chat_id: {self.chat_id}"
 
     def data_to_json(self):
-        data = {"chat_id": self.chat_id, "title": self.title}
-        return data
+        return {
+            "chat_id": self.chat_id,
+            "title": self.title,
+            "teams": [team.data_to_json() for team in self.teams],
+        }
+
+    def add_team(self, team):
+        self.teams.append(team)
+
+    def remove_team(self, team_id: int):
+        team_to_remove = next(
+            (team for team in self.teams if team.team_id == team_id), None
+        )
+        if team_to_remove:
+            self.teams.remove(team_to_remove)
+
+    def has_team(self, team_id: int) -> bool:
+        return any(team.get_team_id() == team_id for team in self.teams)
+
+    def get_team_by_id(self, team_id: int) -> TeamBase:
```

Comment:
But here you didn't use a private method and avoided adding nesting. What's the difference?